02-oop-inheritance/javaScript/downloableMovie.js

```diff
@@ -0,0 +1,59 @@
+function MovieObserver() {
+
+  this.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+  } 
+  this.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+  } 
+}
+
+function Movie() {
+  this.atributes = new Array();
```

Comment:
First of all attributes has a type, also attributes must be an object (my guess) not an array, and finally is you are using arrays we should use [ ] instead of new Array(), we've talked about this last Friday